### PR TITLE
ju/ednx/E-07 Add command "change_domain"

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description
Add command "change_domain" from https://github.com/eduNEXT/edunext-ecommerce/pull/28

### CODE REVIEW

File: ecommerce_extensions/management/commands/change_domain.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        parser.add_argument('suffix_lms_domain', type=str,
+                            nargs='?', default=CHANGE_DOMAIN_DEFAULT_SITE_NAME,
+                            help='Suffix domain appended to LMS domains')
+
+    def handle(self, *args, **options):
+        """
+        This method will iterate over all Sites and SiteConfiguration
+        objects to change Site prod domains to a stage version.
+        """
+        self.suffix_stage_domain = options['suffix_domain']
```
Comment (mariajgrimaldi) [inline ecommerce_extensions/management/commands/change_domain.py:41]: options will always have suffix_domain and suffix_lms_domain? or if one of them is missing this must fail?

File: ecommerce_extensions/management/tests/test_change_domain.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+        with patch.object(Site, "objects") as site_objects_mock:
+            site_objects_mock.all.return_value = sites
+            call_command(
+                "change_domain",
+                suffix_domain=".suffix.com",
+                suffix_lms_domain=".suffix.com"
+            )
+
+            siteconfiguration = site_objects_mock.all()[0].siteconfiguration
```
Comment (mariajgrimaldi) [inline ecommerce_extensions/management/tests/test_change_domain.py:93]: what about  `siteconfiguration.save.assert_called_once()`?

File: ecommerce_extensions/management/commands/change_domain.py (hunk 1 above)
Comment (andrey-canon) [reply ecommerce_extensions/management/commands/change_domain.py:41]: those options have a default value

**Review by mariajgrimaldi — APPROVED** [review]

**Comment by andrey-canon** [comment]
Closed, since this is not necessary due to scripted-recipes implementations

**Pull request closed without merging**